docs/api_docs/kubeblocks-for-redis/redis-cluster-mode.md

```diff
@@ -338,6 +338,12 @@ spec:
 
 ### Restart
 
+:::note
+
+Currently, restarting a Redis Cluster is not supported.
+
+:::
+
 ```yaml
```

Comment:
the yaml example can be removed,  and we will support redis cluster restart in the future

Reply:
> the yaml example can be removed, and we will support redis cluster restart in the future

Fixed.